dts/bindings/iio/afe/current-sense-amplifier.yaml

```diff
@@ -38,6 +38,12 @@ properties:
     description: |
       Amplifier gain divider. The default is <1>. The maximum value is <65535>.
 
+  zero-current-voltage-milli-volts:
+    type: int
+    default: 0
+    description: |
+      The zero current voltage in mV. The default is <0>. The maximum value is <65535>.
```

Comment:
please read https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values and fix this